TagsterWebAPI/src/Tagster.Infrastructure/EF/TagsterDbContext.cs

```diff
@@ -1,22 +1,38 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+using Tagster.Auth.Services;
+using Tagster.Domain.Authorization;
 using Tagster.Domain.Entities;
 
 namespace Tagster.Infrastructure.EF;
 
-public class TagsterDbContext : DbContext
+internal class TagsterDbContext : DbContext
 {
+    private readonly IEnumerable<User> DefaultUsers;
+    private readonly IPasswordService _passwordService;
+
     public DbSet<Tag> Tags { get; set; }
     public DbSet<Profile> Profiles { get; set; }
     public DbSet<User> Users { get; set; }
     public DbSet<RefreshToken> RefreshTokens { get; set; }
 
-    public TagsterDbContext(DbContextOptions options) : base(options)
+    public TagsterDbContext(DbContextOptions options, IPasswordService passwordService) : base(options)
     {
+        _passwordService = passwordService;
+        DefaultUsers = CreateDefaultUsers();
```

Comment:
do we need this call and assignment in ctor? As I can see we use it to generate migration only so calling it from OnModelCreating should be enough. By doing this we can also remove DefaultUsers field declaration.

Reply:
I did small refactor